jbpm-persistence/jbpm-persistence-api/src/main/java/org/jbpm/persistence/api/integration/base/TransactionalPersistenceEventManager.java

```diff
@@ -153,5 +153,6 @@ public void close() {
         }
         
         this.emitter.close();
+        emitter = null;
```

Comment:
maybe it's worth checking whether `this.emitter` is `null` as it might be already closed. wdyt?
```suggestion
        if (this.emitter != null) {
            this.emitter.close();
            emitter = null;
        }
```